### ISSUE TITLE
trying the LinkLocator example gives me this error

### ISSUE THREAD

**Issue by kunalbedi** [trafficgate/linkshare-api#9]
Making one call every 12 second(s) to obey the limit of 5 requests per minute.
API URL:                https://api.rakutenmarketing.com/linklocator/1.0/getMerchByName/Udemy

```
PHP Fatal error:  Uncaught exception 'Exception' with message 'Return data was in an unexpected format.' in C:\Users\kunal\Desktop\linkshare\vendor\linkshare\linkshare-api\src\Api\LinkLocator.php:95
Stack trace:
#0 C:\Users\kunal\Desktop\linkshare\linkshare-api.php(217): Linkshare\Api\LinkLocator->get()
#1 C:\Users\kunal\Desktop\linkshare\linkshare-api.php(81): makeRequest(Object(Linkshare\Api\LinkLocator), Object(Closure), 5, 'C:\\Users\\kunal\\...')
#2 {main}
  thrown in C:\Users\kunal\Desktop\linkshare\vendor\linkshare\linkshare-api\src\Api\LinkLocator.php on line 95

Fatal error: Uncaught exception 'Exception' with message 'Return data was in an unexpected format.' in C:\Users\kunal\Desktop\linkshare\vendor\linkshare\linkshare-api\src\Api\LinkLocator.php:95
Stack trace:
#0 C:\Users\kunal\Desktop\linkshare\linkshare-api.php(217): Linkshare\Api\LinkLocator->get()
#1 C:\Users\kunal\Desktop\linkshare\linkshare-api.php(81): makeRequest(Object(Linkshare\Api\LinkLocator), Object(Closure), 5, 'C:\\Users\\kunal\\...')
#2 {main}
  thrown in C:\Users\kunal\Desktop\linkshare\vendor\linkshare\linkshare-api\src\Api\LinkLocator.php on line 95
```

Environment: PHP 7.1.1 cli

**Comment by ikari7789** [comment]
Could you perhaps add a var_dump before line #95 in `src/Api/LinkLocator.php`  for the `$result` variable and share with me what it gives you? (Please edit out any sensitive information in the result if it exists).

**Comment by kunalbedi** [comment]
Hi Ikari,

Thank you for the response. I have added/attached the var dump of $result at line 93 - 95 below.
Regards
Kunal
[vardump.txt](https://github.com/trafficgate/linkshare-api/files/1466486/vardump.txt)

**Comment by ikari7789** [comment]
Hmmm, seems it isn’t returning the response body but rather the full client itself. Would it be possible to share which dependencies your application is using? There may be a version issue for something.

**Comment by kunalbedi** [comment]
Hi,

Thank you for immediate response.
The zip contains the composer.json via which I installed the api.
The other file is installed.json which I have taken from vendor/composer directory listing all the packages which were installed when I issued **composer require linkshare/linkshare-api**.
Please note that I am running PHP 7.1.1 on windows 10 and tested the things using php in CLI mode.
[Desktop.zip](https://github.com/trafficgate/linkshare-api/files/1466597/Desktop.zip)

Thank you.

**Comment by ikari7789** [comment]
Well, I think I've managed to fix the issue in the branch `feature/fix-response-for-link-locator`. It also doesn't help that the example code is not so great and probably shouldn't have been included in the first place. I cleaned it up a little by separate each different API call into it's own example file. See if that helps you out at all.

**Comment by kunalbedi** [comment]
Hi Andrew,

The link locator old code has started working. I will check some more and will post if I find any more issues.

I thank you for your express action on this and really appreciate it.

Regards
Kunal Bedi

**Comment by andythorne** [comment]
@ikari7789 This is still an issue, `feature/fix-response-for-link-locator` should fix it.

**Comment by ikari7789** [comment]
This should be fixed by release v1.0.9.
